test/lib/test_block.ml

```diff
@@ -25,6 +25,13 @@ let test_mk =
         Mdx.Block.mk ~loc:Location.none ~section:None ~labels
           ~legacy_labels:false ~header ~contents ~errors:[]
       in
+      let expected =
+        Result.map_error
+          (function
```

Comment:
I would assume that ``(fun (`Msg m) -> …)`` would work here as well and be a bit simpler?